src/containers/Tenant/Diagnostics/TenantOverview/Healthcheck/IssuesViewer/IssueTree.tsx

```diff
@@ -28,12 +29,7 @@ const IssueTree = ({issueTree}: IssuesViewerProps) => {
 
         return (
             <div className={b('info-panel')}>
-                <JSONTree
-                    data={info}
-                    search={false}
-                    isExpanded={() => true}
-                    treeClassName={b('inspector')}
-                />
+                <JsonViewer value={unipikaConvert(info)} />
```

Comment:
It looks like we need to remove background here

![Screenshot 2025-02-14 at 11 59 45](https://github.com/user-attachments/assets/f3a8e879-d8c5-4c29-98ce-ca03d1e64fb8)

![Screenshot 2025-02-14 at 11 59 53](https://github.com/user-attachments/assets/0b28ad81-598b-4df3-a7c8-8f3c00b05a64)